### ISSUE TITLE
WCAG 4.1.2 test 10: don't apply to titles?

### ISSUE THREAD

**Issue by jzuidweg** [veraPDF/veraPDF-wcag-algs#96]
[This document](https://github.com/veraPDF/veraPDF-wcag-algs/files/7396935/test-document.pdf) fails **WCAG 4.1.2 test 10** ('_Heading is incorrectly tagged as a paragraph_'). It flags this error on the first line of the document, which is the title.

I had a long discussion with WCAG experts whether it's appropriate to give this error here. For websites, WCAG requires the title to be tagged as `<h1>` and there may only be one `<h1>` in the document. However, for PDF documents this WCAG rule appears to be inappropriate as a reader would not expect to see the title appear as a header of level 1 in the table of contents.

For this reason, WCAG now accepts PDF documents in which the title is not tagged as a `<h1>`. So perhaps we should not generate the error for **WCAG 4.1.2 test 10** on text that was tagged as the title, or clearly intended as title of the document. What do you think?

**Comment by bdoubrov** [comment]
Here is the relevant discussion on this subject: https://www.pdfa.org/how-to-tag-titles-in-pdf-documents/

We'll add the support for `<title>` tag (which exists for the moment only in PDF 2.0) and will suggest using such tag in the situation when we detect the heading as a first (non-empty) paragraph in the document.

**Comment by jzuidweg** [comment]
This issue has been tested and is solved. It can be closed.
